taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+
+      <p>
+        Assuming your TACC username is <code>jdoe</code> and you are affiliated with UT Austin, a <code>scp</code> transfer that pushes <code>my_file.txt</code> from your local computer to the remote secure system would look like this:
+      </p>
+      <pre style="max-width: 1500px"><code><u>localhost$ </u>scp ./my_file.txt jdoe@<kbd>host</kbd>:<kbd>/transfer/directory/path</kbd></code></pre>
+      <blockquote>
+        <p>
+          Note: This command will copy your data file directly to your individualized transfer directory on the remote storage system.
+        </p>
+      </blockquote>
+      <p>
+        <em>If you have not done so already, enter this command in your terminal,
+          replacing the file name, TACC username, and your individualized transfer directory name appropriately.</em>
+      </p>
+      <p>
+        After entering the command, you will be prompted to login to the remote secure system by entering the password associated with your TACC account as well as the token value generated from your TACC token app.
+      </p>
+      <p>
+        The data transfer was successful if your terminal output looks like this:
+      </p>
+      <pre style="max-width: 500px"><samp>my_file.txt     100% ##  #.#          KB/s   ##:##</samp></pre>
+
+      <p>
+        If you wish to learn more about <code>scp</code> and how to synchronize your file transfer, you can do so
+        <a target="_blank" href="https://man7.org/linux/man-pages/man1/scp.1.html">the online <code>man</code> page for <code>scp</code></a>
+        or follow the file transfer section of the user guide for the relevant storage system:
+      </p>
+      <ul>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides/stampede2#transferring-rsync">Stampede2 User Guide: Transferring Files Using <code>scp</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://frontera-portal.tacc.utexas.edu/user-guide/files/#transferring-files-with-rsync">Frontera User Guide: Transferring Files with <code>rsync</code></a>
+        </li>
+        <li>
+          <a target="_blank" href="https://portal.tacc.utexas.edu/user-guides">TACC User Guides (for each system)</a>
+          <small>(select the relevant system for this portal, then search "<code>scp</code>")</small>
+        </li>
+      </ul>
+
+      <h4 id="sftp_transfer">Transfer Data with <code>sftp</code></h4>
+
+      <p>
+        <code>sftp</code>
+        is a file transfer program that allows you to interactively navigate between your local file system and the remote secure system. To transfer a file (ex. <code>my_file.txt</code>) to the remote secure system via <code>sftp</code>, open a terminal on your local computer and navigate to the path where your data file is located.&nbsp;
+      </p>
+      <pre style="max-width: 500px"><code><u>localhost$ </u>cd Documents/portal-data/</code></pre>
+
+      <p>
```

Comment:
...Austin, an <code>sftp</code> transfer that pushes...

...from the current directory of your local computer...